Amazon.IonObjectMapper.Test/IonObjectSerializerTest.cs

```diff
@@ -494,10 +488,12 @@ public void SerializesObjectsWithCustomSerializers()
 
             Assert.IsTrue(serialized.ContainsField("engine"));
             var engine = serialized.GetField("engine");
-            Assert.IsTrue(engine.ContainsField("manufactureDate"));
+            Assert.IsTrue(engine.ContainsField("Cylinders"));
+            Assert.AreEqual(3, engine.GetField("Cylinders").IntValue);
+            Assert.IsTrue(engine.ContainsField("ManufactureDate"));
             Assert.AreEqual(
-                TestObjects.honda.Engine.ManufactureDate.AddDays(1), 
-                engine.GetField("manufactureDate").TimestampValue.DateTimeValue);
+                DateTime.Now.ToString(), 
```

Comment:
This test was checking that the custom `DateTime` serializer (which simply adds a day to the date) is used for `DateTime` objects but your choice to implement a custom serializer for `Engine` messes with this test (and potentially other tests that also use `Engine`).
I think it's better to define an entirely new class for testing the custom serialization so all the other tests can be left alone.

Reply:
Make sense. Created a new test object.